cmd/wfx-viewer/output/plantuml/plantuml_test.go

```diff
@@ -0,0 +1,19 @@
+package plantuml
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/siemens/wfx/workflow/dau"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenerate(t *testing.T) {
+	buf := new(bytes.Buffer)
+	gen := NewGenerator()
+	err := gen.Generate(buf, dau.DirectWorkflow())
+	require.NoError(t, err)
+	expected := "@startuml\nstate INSTALL as \"<color:black>INSTALL</color>\" #00cc00: instruct client to start installation\nstate INSTALLING as \"<color:black>INSTALLING</color>\" #00cc00: installation progress update from client\nstate INSTALLED as \"<color:black>INSTALLED</color>\" #00cc00: client signaled installation success\nstate ACTIVATE as \"<color:black>ACTIVATE</color>\" #00cc00: instruct client to start activation\nstate ACTIVATING as \"<color:black>ACTIVATING</color>\" #00cc00: client activates update\nstate ACTIVATED as \"<color:black>ACTIVATED</color>\" #4993dd: client signaled activation success\nstate TERMINATED as \"<color:black>TERMINATED</color>\" #9393dd: client aborted update with error\nINSTALL --> INSTALLING: CLIENT\nINSTALL --> TERMINATED: CLIENT\nINSTALLING --> INSTALLING: CLIENT\nINSTALLING --> TERMINATED: CLIENT\nINSTALLING --> INSTALLED: CLIENT\nINSTALLED --> ACTIVATE: WFX [IMMEDIATE]\nACTIVATE --> ACTIVATING: CLIENT\nACTIVATE --> TERMINATED: CLIENT\nACTIVATING --> ACTIVATING: CLIENT\nACTIVATING --> TERMINATED: CLIENT\nACTIVATING --> ACTIVATED: CLIENT\nlegend right\n  | Color | Group | Description |\n  | <#00cc00> | OPEN | regular workflow-advancing states |\n  | <#4993dd> | CLOSED | a successful update's terminal states |\n  | <#9393dd> | FAILED | a failed update's terminal states |\n  | <#000000> |  | The state doesn't belong to any group. |\nendlegend\n@enduml\n"
```

Comment:
the presence of other tests of the kind shows that approval tests might be beneficial

Reply:
Yep, changed.